src/components/ObsDetails/DetailsTab.js

```diff
@@ -0,0 +1,157 @@
+
+  const displayQualityGradeOption = option => {
+    const qualityGrade = observation?.quality_grade;
+    const labelClassName = ( qualityGrade === option ) ? "font-bold" : "";
+
+    return (
+      <View className="flex-col space-y-[8px]">
+        <QualityGradeStatus qualityGrade={option} />
+        <Body4 className={labelClassName}>{_.startCase( _.camelCase( t( option ) ) )}</Body4>
+      </View>
+    );
+  };
+
+  return (
+    <>
+      {observation.description && (
+        <>
+          <View className={sectionClass}>
+            <Heading4 className={headingClass}>{t( "NOTES" )}</Heading4>
+            <Body2>{observation.description}</Body2>
+          </View>
+          <Divider />
+        </>
+      )}
+
+      <View className="flex-row justify-between items-center mt-[8px] mx-[15px]">
+        <Heading4>{t( "LOCATION" )}</Heading4>
+        <KebabMenu
+          visible={locationKebabMenuVisible}
+          setVisible={setLocationKebabMenuVisible}
+        >
+          <Menu.Item
+            onPress={async ( ) => {
+
+            }}
+            title={t( "Share-location" )}
+          />
+          <Menu.Item
+            onPress={async ( ) => {
+
+            }}
+            title={t( "Copy-coordinates" )}
+          />
+        </KebabMenu>
+      </View>
+      {isOnline
+        ? (
+          <Map
+            obsLatitude={observation.latitude}
+            obsLongitude={observation.longitude}
+            mapHeight={230}
+          />
+        ) : (
+          <View className="h-16 items-center justify-center">
+            <IconMaterial
+              name="wifi-off"
+              size={30}
+              accessibilityRole="image"
+              accessibilityLabel={t( "Location-map-unavailable-without-internet" )}
+            />
+          </View>
+        )}
+      <View className={`mt-[11px] ${sectionClass}`}>
+        <ObservationLocation observation={observation} details />
+      </View>
+      <Divider />
+
+      <View className={sectionClass}>
+        <Heading4 className={headingClass}>{t( "DATE" )}</Heading4>
+        <DateDisplay
+          classNameMargin="mb-[12px]"
+          label={t( "Observed" )}
+          dateString={checkCamelAndSnakeCase( observation, "timeObservedAt" )}
+        />
+        <DateDisplay
+          label={t( "Uploaded" )}
+          dateString={checkCamelAndSnakeCase( observation, "createdAt" )}
+        />
+      </View>
+      <Divider />
+
+      <View className={`${sectionClass} flex-col`}>
+        <Heading4 className={headingClass}>{t( "DATA-QUALITY" )}</Heading4>
+        <View className="space-y-[15px]">
+          <View className="flex-row justify-around">
+            {displayQualityGradeOption( "casual" )}
+            {displayQualityGradeOption( "needs_id" )}
+            {displayQualityGradeOption( "research" )}
+          </View>
+          <Body4>
+            {t( "This observation needs more identifications to reach research grade" )}
```

Comment:
I don't think we can use full sentences with spaces as keys, also I don't see this in strings.ftl.

Reply:
you're right. I've added the strings to strings.ftl. Only the needs-id string was included in the design docs so I added some placeholder strings for research and casual grade for now.